kmk/extensions/peg_rgb_matrix.py

```diff
@@ -37,6 +38,24 @@ def generate_led_map(
         keys = [key_color] * number_of_keys
         underglow = [underglow_color] * number_of_underglow
         print(f'Rgb_matrix_data(keys={keys},\nunderglow={underglow})')
+        return Rgb_matrix_data(keys, underglow)
+
+
+class Rgb_matrix_layers:
+    def __init__(self, layers):
+        self.layers = layers if layers else []
+
+    @staticmethod
+    def generate_layer(number_of_leds, layer_color):
```

Comment:
One of these needs removed. Having two methods with the same name is failing the test.